### PR TITLE
Add supported token list for ethereum

### PR DESCRIPTION
**Pull request by himanshugarg06**

### CODE REVIEW

File: docs/supportedchains/supportedTokens.md [hunk 1]
```diff
@@ -5,9 +5,9 @@ sidebar_position: 2
 
 # ERC20 Gas Payments
 
-Our Paymaster not only allows you to sponsor gasless tokens for your users but also allows for users to pay for tokens in supported ERC20 tokens. Below are supported tokens grouped by chain.
+Our Paymaster not only allows you to sponsor gasless tokens for your users but also allows users to pay for tokens in supported ERC20 tokens. Below are supported tokens grouped by chain.
```
Comment (divyaN73) [inline docs/supportedchains/supportedTokens.md:8]: "Our Paymaster not only allows you to sponsor gasless tokens for your users"

it should be

"Our Paymaster not only allows you to sponsor transactions for your users"...

File: docs/supportedchains/supportedTokens.md [hunk 2]
```diff
@@ -5,9 +5,9 @@ sidebar_position: 2
 
 # ERC20 Gas Payments
 
-Our Paymaster not only allows you to sponsor gasless tokens for your users but also allows for users to pay for tokens in supported ERC20 tokens. Below are supported tokens grouped by chain.
+Our Paymaster not only allows you to sponsor transactions for your users but also allows users to pay for transactions using ERC20 tokens. Below are supported tokens list grouped by chain.
 
-Please note that we will be reviewing DEX activitiy on each network to determine additional ERC20 token supoort on an ongoing basis.
+Please note that we will be reviewing DEX activity on each network to determine additional ERC20 token support on an ongoing basis.
```
Comment (livingrockrises) [inline docs/supportedchains/supportedTokens.md:10]: we should add

Want to add a token? Please [contact us](mailto:support@biconomy.io) and we'll see what we can do.

do confirm contact email

**Comment by livingrockrises** [comment]
I think the pair column is unnecessary

**Comment by livingrockrises** [comment]
I think it also makes sense to add testnet tokens. later we can mention more about faucets or getting in touch to get tokens

**Comment by livingrockrises** [comment]
we do support so manyy tokens on BSC mainnet. I expect list to be longer. can you double check in config file and with @divyaN73

**Review by divyaN73 — DISMISSED** [review]

**Comment by divyaN73** [comment]
> we do support so manyy tokens on BSC mainnet. I expect list to be longer. can you double check in config file and with @divyaN73

cross checked on growthbook too, it showing 32 tokens approx. I'll get this checked from shurti next week

File: docs/supportedchains/supportedTokens.md [hunk 3]
```diff
@@ -5,9 +5,9 @@ sidebar_position: 2
 
 # ERC20 Gas Payments
 
-Our Paymaster not only allows you to sponsor gasless tokens for your users but also allows for users to pay for tokens in supported ERC20 tokens. Below are supported tokens grouped by chain.
+Our Paymaster not only allows you to sponsor transactions for your users but also allows users to pay for transactions using ERC20 tokens. Below are supported tokens list grouped by chain.
```
Comment (NunoFilipeSantos) [inline docs/supportedchains/supportedTokens.md:8]:
```suggestion
Our Paymaster not only allows you to sponsor transactions for your users but also allows users to pay for transactions using ERC20 tokens. Below are the supported tokens list grouped by chain.
```

**Review by NunoFilipeSantos — CHANGES_REQUESTED** [review]
"the" missing.

**Comment by himanshugarg06** [comment]
> I think it also makes sense to add testnet tokens. later we can mention more about faucets or getting in touch to get tokens

Will create a new task for these.

**Pull request merged**